internal/utils.go

```diff
@@ -61,18 +59,50 @@ func getPolicyTemplate(policyConf *policyConfig) (
 				continue
 			}
 
-			manifests = append(manifests, *manifestFile...)
+			manifestFiles = append(manifestFiles, *manifestFile...)
+		}
+
```

Comment:
All rest logic in getManifests and getPolicyTemplate looks good to me. After these refactorizations are merged, I also need to update my pr accordingly.

Thanks for so detailed comments!